x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/policy_settings_form.test.tsx

```diff
@@ -50,6 +50,17 @@ describe('Endpoint Policy Settings Form', () => {
     render = () => (renderResult = mockedContext.render(<PolicySettingsForm {...formProps} />));
   });
 
+  it('should show the event merging banner for 8.16', () => {
+    render();
+    expect(renderResult.getByTestId('eventMergingCallout')).toBeTruthy();
+  });
+
+  it('should not show the event merging banner after it has been dismissed', () => {
```

Comment:
is the dismiss of the banner saved to local storage so that it does not show up again? if so, can you add tests for that?

Reply:
yes, it's persisted, and here's the test for it:
7afb2182e034bb58cce50fe737b4d27f9bc8feb0